### PR TITLE
Initial work to cast the toggle string as a boolean as a workaround for an arcgis problem with booleans

### PR DESCRIPTION
**Pull request by drewpotter**
This PR is the initial work for :

- A workaround for an ArcGIS issue which is the lack of support for boolean types. So we now cast the string of either `t` or `f` to a boolean using the `::boolean` cast in Postgres/PostGIS. 
- Renaming the `action` column to `validate_action` which was necessary as the previous name was causing problems on Carto (due to PostgreSQL using `action` as a reserved word).
- Updated the UNEP-WCMC logo on the footer on the landing home page.
- Added `iso3` field to `countries` table, this can be removed if it is not really necessary. I just prefer using ISO3 personally and wanted to transition over to using ISO3. But happy to remove this work. It is necessary to drop the `countries` and re-seed the database to test that.
- Fixed all broken tests

### CODE REVIEW

**Comment by drewpotter** [comment]
Many thanks!

File: lib/modules/carto_db/proxy/map.rb [hunk 1]
```diff
@@ -8,6 +8,8 @@ def self.create habitat, opts={}
       body: mapconfig_json(sql, cartocss),
       headers: {'Content-Type' => 'application/json'}
     )
+
+    Rails.logger.info "carto_response: #{carto_response.inspect}"
```
Comment (Levia) [inline lib/modules/carto_db/proxy/map.rb]: Could you please get rid of the loggers if unnecessary?

File: lib/modules/carto_db/proxy/map.rb [hunk 2]
```diff
@@ -16,7 +18,7 @@ def self.create habitat, opts={}
   def self.extract_tiles_url response
     info = JSON.parse(response)
     tiles_path = URI.escape("/api/v1/map/#{info["layergroupid"]}/{z}/{x}/{y}.png")
-
+    Rails.logger.info "LAYERGROUPID: #{info['layergroupid']}"
```
Comment (Levia) [inline lib/modules/carto_db/proxy/map.rb]: Same here

File: lib/modules/carto_db/proxy/map.rb [hunk 3]
```diff
@@ -38,6 +40,7 @@ def self.sql habitat, country, where
     parts = "SELECT * FROM #{source(habitat)}"
     parts << " WHERE country_id = '#{country.iso}'"
     parts << " AND #{where}" if where
+    Rails.logger.info parts
```
Comment (Levia) [inline lib/modules/carto_db/proxy/map.rb]: Same here

**Comment by Levia** [comment]
@drewpotter Happy with this when the loggers are removed if unnecessary !

**Comment by drewpotter** [comment]
@Levia I have removed the loggers as requested!

**Comment by drewpotter** [comment]
Thanks @Levia with regards to the two points.

1. I think the frontend will be looked at separately, so it should be fixed in a later PR.

2. I think this must be a WIP and since there is no documentation I think we can ignore that for now. Please advise if that makes sense?

**Pull request merged**